roles/filetree_create/templates/current_workflow_job_templates.j2

```diff
@@ -105,8 +105,8 @@ controller_workflows:
                               | default(query(controller_api_plugin, current_workflow_job_templates_asset_value.related.survey_spec,
                                               host=controller_hostname, oauth_token=controller_oauthtoken, verify_ssl=controller_validate_certs)[0])
                               | from_yaml | to_nice_yaml(indent=2,width=500, sort_keys=False) | regex_replace("\n\n[ ]*", "\\\\n")
-                              | indent(width=6, first=False) | replace("'{{", "!unsafe \'{{") | replace("^$", "") | replace("$encrypted$", "\'\'")
```

Comment:
`replace("$encrypted", "\'\'")` is needed to solve issue #8... isn't it needed anymore?